zhusuan/distributions/multivariate.py

```diff
@@ -29,6 +33,158 @@
 ]
 
 
+class MultivariateNormalTriL(Distribution):
+    """
+    The class of multivariate normal distribution, where covariance is
+    parameterized with the lower triangular matrix in Cholesky decomposition,
+
+        .. math :: L \\text{s.t.} LL^T = \Sigma.
+
+    See :class:`~zhusuan.distributions.base.Distribution` for details.
+
+    :param mean: An N-D `float` Tensor. mean[..., i_{N-1}, :] corresponds to a
```

Comment:
N-D (N >= 1) `float` Tensor of shape ``[..., n_dims]``. Each slice `[i, j,..., k, :]` represents the mean of a single multivariate normal distribution.
similar applies to `cov_tril`.